sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -368,3 +368,12 @@ def create_properties_from_dict_if_needed(properties, sb_resource_type):
                 sb_resource_type.__name__
             )
         )
+
+
+def override_properties_with_keyword_arguments(properties, **kwargs):
+    # type: (PropertiesType, Any) -> None
+    if not kwargs:
+        return
+    for key, _ in kwargs:
+        if key in properties:
```

Comment:
that is true, I'm doing double check here, my major thinking is that the method itself should be an safe update for properties, not polluting properties objects with new keys so the method could be reused.

the _management_client.py one is for popping keys to avoid azure-core error